### PR TITLE
Output VMSS cloud config settings.

### PR DESCRIPTION
**Pull request by JunSun17**
Output VMSS cloud config settings for AKS to consume.

### CODE REVIEW

**Comment by jackfrancis** [comment]
@JunSun17 What's the functional purpose of this change?

File: parts/iaasoutputs.t [hunk 1]
```diff
@@ -25,4 +25,13 @@
     "primaryAvailabilitySetName": {
         "type": "string",
         "value": "[variables('primaryAvailabilitySetName')]"
+    },
+    "primaryScaleSetName": {
+        "type": "string",
+        "value": "[variables('primaryScaleSetName')]"
+    },
+    "vmType": {
```
Comment (weinong) [inline parts/iaasoutputs.t]: we really dont need this

Comment (JunSun17) [reply parts/iaasoutputs.t]: This change is originally target to pass the cloud config settings of vmType and primaryScaleSetName to AKS (in the way of exposing it to ARM output).

I will see if we can build these strings in AKS for its cloud provider config.

File: parts/k8s/kubernetesmastervars.t [hunk 2]
```diff
@@ -359,11 +359,11 @@
     "nsgID": "[resourceId('Microsoft.Network/networkSecurityGroups',variables('nsgName'))]",
 {{if not AnyAgentUsesVirtualMachineScaleSets}}
     "primaryAvailabilitySetName": "[concat('{{ (index .AgentPoolProfiles 0).Name }}-availabilitySet-',variables('nameSuffix'))]",
-    "primaryScaleSetName": "''",
+    "primaryScaleSetName": "",
```
Comment (jackfrancis) [inline parts/k8s/kubernetesmastervars.t]: What is the purpose of these changes (from literal `''` to empty string)?

Comment (JunSun17) [reply parts/k8s/kubernetesmastervars.t]: If it is set to "''", it will not be an empty string in the output. the output will be ''. We need an empty string as the result.
